packages/client/lib/commands/HEXPIRE.ts

```diff
@@ -0,0 +1,43 @@
+import { Command, RedisArgument } from '../RESP/types';
+import { pushVariadicArgument } from './generic-transformers';
+
+/**
+ * @readonly
+ * @enum {number}
+ */
+export const HASH_EXPIRATION = {
+  /** @property {number} */
```

Comment:
?!

Reply:
I was trying to write jdoc I guess?  probably don't know what I'm doing with it.